Sming/Components/Network/src/Network/SmtpClient.cpp

```diff
@@ -296,6 +297,12 @@ void SmtpClient::sendMailHeaders(MailMessage* mail)
 		mail->stream = std::make_unique<QuotedPrintableOutputStream>(mail->stream.release());
 	}
 
+	if(!mail->headers.contains(F("Message-ID"))) {
```

Comment:
Should we add `Message-ID` to `HTTP_HEADER_FIELDNAME_MAP` ?

Reply:
https://en.wikipedia.org/wiki/Message-ID is not really related or used in HTTP.